lxd/db/errors.go

```diff
@@ -2,9 +2,19 @@ package db
 
 import (
 	"fmt"
+	"net/http"
+
+	"github.com/canonical/go-dqlite/v3/driver"
+	"github.com/mattn/go-sqlite3"
 )
 
 var (
 	// ErrNoClusterMember is used to indicate no cluster member has been found for a resource.
 	ErrNoClusterMember = fmt.Errorf("No cluster member found")
 )
+
+// SmartErrors are used to return more appropriate errors to the caller.
+var SmartErrors = map[int][]error{
```

Comment:
So in case of `microcluster/v2` we create our own `SmartErrors` using `go-dqlite/v2`?

Reply:
And the call `Init()` using our own to prevent importing the `v3`.

Reply:
yeah thats right